source/numerics.tex

```diff
@@ -9648,13 +9679,16 @@
 
 \pnum
 \returns
-\[ \mathsf{L}_n^m(x) =
-   (-1)^m \frac{\mathsf{d} ^ m}{\mathsf{d}x ^ m} \, \mathsf{L}_{n+m}(x)
-   \text{ ,\quad for $x \ge 0$,} \]
+The value given by \eqref{sf.cmath.assoc.laguerre},
```

Comment:
Maybe we need to say more here. The formula isn't a "value" as such. Maybe "the value L_n^m(x), where L_n^m is given by Formula ..."?

Reply:
Maybe we can omit the "The value given by" and just say the function returns "L_n^m(x)` ..." and then the rest from your suggestion.

Reply:
Updated; please have a look.